.github/workflows/ci.yaml

```diff
@@ -80,12 +80,11 @@ jobs:
         # https://github.com/instrumenta/kubernetes-json-schema is no
         # longer updated
         k8s:
-          # - v1.14.10 (deprecated with kind 1.16)
-          - v1.16.15
           - v1.18.20
-          - v1.22.9
-          - v1.24.2
-          # v1.25.2 (when agent 7.40 is out)
+          - v1.22.17
+          - v1.24.17
+          - v1.25.15
+          - v1.26.10
```

Comment:
For this I am not sure what the implications are for the agent - I'll let another reviewer comment for these

Reply:
This is also pretty old / crusty, so may need to be updated again.

Reply:
I'm happy to plugin whatever values you all would like here.

I'm not currently using Datadog, but I'm happy to do some small changes if it will help get this shipped.

Reply:
https://github.com/yannh/kubernetes-json-schema
these are the current schema versions available.

I can add 1.27 / 1.28 if desired, and / or update to `v1.25.16` and `v1.26.13`.

Note that `.github/workflows/go-test.yaml` also has a matrix of Kube versions. If someone wants to provide a list of what versions are wanted for either / both, I can update them, but I'll probably default to not touching more things than necessary.

Reply:
If we can we would like to keep Kubernetes 1.16 in our tests. 
Unfortunately we still have some user with this version, but this version is the pre-version of several resources deprecation in the API server.

Reply:
If `kubeconform` is not available for 1.16, maybe we can keep kubeval for 1.16, and use `kubeconform` for the more ressent versions.  WDYT